pkg/syncer/syncer.go

```diff
@@ -72,6 +74,31 @@ func StartSyncer(ctx context.Context, upstream, downstream *rest.Config, resourc
 	go specSyncer.Start(ctx, numSyncerThreads)
 	go statusSyncer.Start(ctx, numSyncerThreads)
 
+	// TODO(marun) Report pcluster connectivity to kcp
+	kcpClusterClient, err := kcpclient.NewClusterForConfig(upstream)
+	if err != nil {
+		return err
+	}
+	kcpClient := kcpClusterClient.Cluster(kcpClusterName)
+	workloadClustersClient := kcpClient.WorkloadV1alpha1().WorkloadClusters()
+	go wait.UntilWithContext(ctx, func(ctx context.Context) {
+		// TODO(marun) Is it necessary to ensure exponential backoff, or will the client backoff automatically?
+		var heartbeatTime time.Time
+		_ = wait.PollImmediateInfiniteWithContext(ctx, 1*time.Second, func(ctx context.Context) (bool, error) {
```

Comment:
don't think there is an automatic backoff

Reply:
Added a todo.